cardano-cli/src/Cardano/CLI/EraBased/Commands/Governance/Actions.hs

```diff
@@ -26,13 +21,37 @@ import           Data.Word
 data GovernanceActionCmds era
   = GovernanceActionCreateConstitution
       (ConwayEraOnwards era)
-      EraBasedNewConstitution
+      Ledger.Network
```

Comment:
It would be better to use record constructors here I think. Makes deconstructing types easier down the road.